kogito-codegen-modules/pom.xml

```
@@ -15,15 +15,35 @@
 
     <name>Kogito :: Codegen Modules</name>
 
-    <modules>
-        <module>kogito-codegen-core</module>
-        <module>kogito-codegen-api</module>
-        <module>kogito-codegen-processes</module>
-        <module>kogito-codegen-rules</module>
-        <module>kogito-codegen-decisions</module>
-        <module>kogito-codegen-predictions</module>
-        <module>kogito-codegen-integration-tests</module>
-        <module>kogito-codegen-sample</module>
-    </modules>
+    <profiles>
+        <profile>
+        <id>default</id>
+        <activation>
+            <activeByDefault>true</activeByDefault>
+        </activation>
+        <modules>
+            <module>kogito-codegen-core</module>
+            <module>kogito-codegen-api</module>
+            <module>kogito-codegen-processes</module>
+            <module>kogito-codegen-rules</module>
+            <module>kogito-codegen-decisions</module>
+            <module>kogito-codegen-predictions</module>
+            <module>kogito-codegen-integration-tests</module>
+            <module>kogito-codegen-sample</module>
+        </modules>
+        </profile>
+        <profile>
+        <id>prod</id>
+        <activation>
+            <property>
+                <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```